src/Core/MainLoop.js

```diff
@@ -153,6 +168,16 @@ MainLoop.prototype._update = function _update(view, updateSources, dt) {
             view.execFrameRequesters(MAIN_LOOP_EVENTS.AFTER_LAYER_UPDATE, dt, this._updateLoopRestarted, geometryLayer);
         }
     }
+
+    if (context.distance.min != Infinity) {
```

Comment:
could be a camera function,` camera.adjustNearFar(min, max)`

Reply:
Yeah, but in this case it would be accessible to anyone. Since it's an internal itowns thing, I prefer to keep this private.

Reply:
Then give me an example please :)